### PR TITLE
Manage Kapacitor Flux tasks in Chronograf

### LINKED ISSUE #2736: Manage Kapacitor Flux tasks in Chronograf
**PR:** https://github.com/influxdata/chronograf/pull/5767

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2736 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/chronograf/v1.9/guides/advanced-kapacitor.md [hunk 1]
```diff
@@ -1,73 +1,77 @@
...
-#### Step 2: Enter a different duration
-
-The minimum allowable duration is one hour (`1h`) and the maximum is infinite (`INF`).
-See the InfluxDB documentation for the list of [acceptable duration units](/{{< latest "influxdb" "v1" >}}/query_language/spec/#duration-units).
-
-#### Step 3: Click the green check mark to save your changes
-
-InfluxDB only keeps data in the `chronograf` database that fall within that new duration; the system automatically deletes any data with timestamps that occur before the duration setting.
+1.  Click **{{< icon "crown" >}} InfluxDB Admin** in the left navigation bar.
+2.  Hover over the retention policy list of the `chronograf` and click **Edit**
```
Comment (noramullen1) [inline content/chronograf/v1.9/guides/advanced-kapacitor.md:35]: Is there a word missing here? "of the `chronograf` something"?

File: content/chronograf/v1.9/guides/advanced-kapacitor.md [hunk 2]
```diff
@@ -1,73 +1,77 @@
...
+### Manage Kapacitor TICKscripts
 
-## TICKscript management
+Chronograf lets you manage Kapacitor TICKscript tasks created in Kapacitor or in
+Chronograf when [creating a Chronograf alert rule](/chronograf/v1.9/guides/create-alert-rules/).
 
-Chronograf creates Kapacitor tasks using the information that you provide on the Rule Configuration page.
-It uses that information to communicate with Kapacitor and populate Chronograf alert pages.
-Pre-existing tasks, or [TICKscripts](/{{< latest "kapacitor" >}}/tick/), that you created and enabled on your Kapacitor instance without using Chronograf, have limited functionality in the user interface.
+To manage Kapacitor TICKscript tasks in Chronograf, click on
```
Comment (noramullen1) [inline content/chronograf/v1.9/guides/advanced-kapacitor.md:55]:
```suggestion
To manage Kapacitor TICKscript tasks in Chronograf, click
```

File: content/chronograf/v1.9/guides/advanced-kapacitor.md [hunk 3]
```diff
@@ -1,73 +1,77 @@
...
-* Enable and disable pre-existing tasks on the Alert Rules page (this is equivalent to the `kapacitor enable` and `kapacitor disable` commands)
-* Delete pre-existing tasks the Alert Rules page (this is equivalent to the `kapacitor delete tasks` command)
+### Manage Kapacitor Flux tasks
+**Kapacitor 1.6+** supports Flux tasks.
+Chronograf lets you view and manage Kapacitor Flux tasks.
 
-You cannot edit pre-existing tasks on the Chronograf Alert Rules page.
-The `mytick` task in the image below is a pre-existing task; its name appears on the Alert Rules page but you cannot click on it or edit its TICKscript in the interface.
-Currently, you must manually edit your existing tasks and TICKscripts on your machine.
+To manage Kapacitor Flux tasks in Chronograf, click on
```
Comment (noramullen1) [inline content/chronograf/v1.9/guides/advanced-kapacitor.md:70]:
```suggestion
To manage Kapacitor Flux tasks in Chronograf, click
```

**Review by noramullen1 — APPROVED** [review]

File: content/chronograf/v1.9/guides/advanced-kapacitor.md (hunk 1 above)
Comment (sanderson) [inline content/chronograf/v1.9/guides/advanced-kapacitor.md:35]:
```suggestion
2.  Hover over the retention policy list of the `chronograf` database and click **Edit**
```

Comment (sanderson) [reply content/chronograf/v1.9/guides/advanced-kapacitor.md]: Yep, should say "`chronograf` database." Fixed it.

**Pull request merged**